pkg/diff/diff.go

```diff
@@ -77,6 +86,13 @@ func DiffUnstructureds(x, y unstructured.Unstructured, opts ...Option) (*Result,
 }
 
 func (d *differ) diff(path []PathStep, vx, vy reflect.Value) error {
+	for _, ignorePathPrefix := range d.ignorePathPrefixs {
```

Comment:
Please add new test cases to ensure that this works as we expected.

Reply:
Ok!

Reply:
I thought that ignore in result.addNode is the simplest and most solid way.
I think that if we add such an implementation in the future, there will be an ignore leakage, because there are cases like diffSlice where addNode is done without going through the diff function.
However, if I have the ignore option in the differ struct, I cannot ignore in the result.addNode. What do you think?

Reply:
Implementation is possible by providing a wrapper function.

Reply:
I implemented it and it went PANIC, so I'm going to proceed the way I've been doing it.